### PR TITLE
[Hotfix] Fix ownership of the log file

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

This PR introduces a fix the for ownership of the log file

### CODE REVIEW

File: egi-check-in-validator.spec [hunk 1]
```diff
@@ -45,11 +45,14 @@ rm -rf $RPM_BUILD_ROOT
 %files -f INSTALLED_FILES
 %{_sysconfdir}/%{name}/config/egi-check-in-validator.ini
 %{_sysconfdir}/%{name}/config/logger.ini
-%{_localstatedir}/log/%{name}/egi-check-in-validator.log
+%dir %attr(0644, condor, condor) %{_localstatedir}/log/%{name}
+%attr(0644,condor,condor) %{_localstatedir}/log/%{name}/egi-check-in-validator.log
```
Comment (NicolasLiampotis) [inline egi-check-in-validator.spec]: We can remove the lines about creating the log files if the default is to use syslog

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
To avoid having to handle different user & group ownership depending on whether the plugin is used by HTCondor or ARC-CE sites I suggest that we change the log configuration to default to syslog only instead of syslog and file. If a site admin prefers syslog+file or just file they can modify `logger.ini` accordingly. So I'd propose the following changes:
1.  Change the default logger.ini installed by the RPM
2. Adjust the logging section of README to describe the new default behaviour (syslog only)
3. Remove the lines from the RPM that create the log file

**Comment by nikosev** [comment]
> 

I have made the required changes

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**